src/client/components/OtherOptions/index.js

```diff
@@ -28,7 +28,9 @@ class OtherOptions extends React.Component {
   render() {
     return (
       <div className="other-options">
-        <button className="option-button">Home</button>
+        <button className="option-button" onClick={function() {
```

Comment:
use Link component instead from the react-router-dom package
`<Link to="/feed">home</Link>`